optimade/models/jsonapi.py

```diff
@@ -137,8 +137,11 @@ class BaseResource(BaseModel):
 
 
 class RelationshipLinks(BaseModel):
-    """A resource object **MAY** contain references to other resource objects (\"relationships\").
-    Relationships may be to-one or to-many. Relationships can be specified by including a member in a resource's links object."""
+    """ A resource object **MAY** contain references to other resource objects (\"relationships\").
```

Comment:
extra space
```suggestion
    """A resource object **MAY** contain references to other resource objects (\"relationships\").
```

Reply:
For similar reasons to your `cleandoc` comment, I don't think these matter as this one is a class docstring. I prefer to keep the space but perhaps its better not to for consistency.